xLights/BitmapCache.cpp

```diff
@@ -601,7 +601,7 @@ wxBitmapBundle BitmapCache::GetModelGroupIcon() {
     return CreateBitmapBundleFromXPMs(16, "ModelGroup", {model_16, model_64, model_64, model_64, model_64});
 }
 wxBitmapBundle BitmapCache::GetFPPIcon() {
```

Comment:
The former call is for a string terminated with \0, but this fpp_app_icon_svg does not end in \0.  Drives the autodebugger nuts.